### PR TITLE
[ERE-948] Registered Clinician - Creating a ‘temporary duplicate’ record

### PR DESCRIPTION
**Pull request by ScriptSmith**

### CODE REVIEW

**Comment by ScriptSmith** [comment]
TODO: Email notifications

File: mnd/mnd/models.py [hunk 1]
```diff
@@ -205,3 +205,8 @@ class CarerRegistration(models.Model):
     registration_ts = models.DateTimeField(null=True, blank=True)
 
     objects = CarerRegistrationManager()
+
+
+class DuplicatePatient(models.Model):
+    patient = models.OneToOneField(Patient, related_name='duplicate_patient', on_delete=models.CASCADE)
+    is_duplicate = models.BooleanField(blank=False, null=False, default=False)
```
Comment (ScriptSmith) [inline mnd/mnd/models.py:213]: Considered only having a `patient` field but I think this is the best way to do it given how demographics fields currently work

**Comment by ScriptSmith** [comment]
https://github.com/eresearchqut/trrf/pull/289

File: mnd/mnd/registry/patients/mnd_admin_forms.py [hunk 2]
```diff
@@ -284,3 +287,20 @@ def _clean_fields(self):
...
+
+
+class DuplicatePatientForm(PrefixedModelForm):
+    is_duplicate = forms.BooleanField(label=_("Mark this patient as a potential duplicate"), required=False)
+
+    class Meta:
+        model = DuplicatePatient
+        fields = ('is_duplicate',)
+
+    def save(self, commit=True):
```
Comment (ScriptSmith) [inline mnd/mnd/registry/patients/mnd_admin_forms.py]: Is it better to use signals here?

Comment (alintreznai) [reply mnd/mnd/registry/patients/mnd_admin_forms.py]: If this is the single place when the notification is triggered then probably not.

**Pull request merged**